app/models/tasks/schedule_hearing_task.rb

```
@@ -93,15 +93,17 @@ def update_from_params(params, current_user)
   end
 
   def available_actions(user)
+    hearing_admin_actions = available_hearing_admin_actions(user)
```

I *think* we don't want change hearing disposition tasks to be available from the schedule hearing task yet. This parent hearing task won't have any active disposition tasks, so I think the logic will just fail silently in create_change_hearing_disposition_task_and_complete_children.